src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -59,7 +80,9 @@ export default class Draw extends Module {
 	private PLAY: number[] = []
 	private ARCANA_STATUSES: number[] = []
 
+
 	protected init() {
+
 		PLAY.forEach(actionKey => {
```

Comment:
nit and maybe not this pr scope but this can just be a `map()`
```js
this.PLAY = PLAY.map(actionKey => this.data.actions[actionKey].id)
```

Reply:
Makes sense, I'll make that change.